report_thesis/src/sections/background/linear_and_regularization_models/enet.tex

```diff
@@ -0,0 +1,15 @@
+\subsubsection{Elastic Net Regression (ENet)}
+Despite \gls{lasso} regression's ability to perform feature selection and its effectiveness in settings with multicollinearity, it can struggle when the dataset contains globally highly correlated features.
+\gls{lasso} tends to select only one feature from a group of highly correlated features and ignores the others.
```

Comment:
Cite `zou_regularization_2005` here as well? Just to cover entirely. From what I can tell, that's where this claim is from.

Reply:
This is implied, so I think we should find an additional source to further back up this claim. I propose this paper:
https://people.ee.duke.edu/~lcarin/lukas-sara-peter.pdf

Specifically, this paragraph:
![image](https://github.com/chhoumann/thesis-chemcam/assets/57709490/181dee47-52c8-4672-a2d9-e3745401c675)

Thoughts?

Reply:
I'm not sure why we'd need another one when Zou makes the case:
![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/d4dab854-0cc1-405f-9886-e21c0f5e67ea)

I just wanted to cite for it to be explicit.

Reply:
But I see that you added it, so it's fine?